third-party/messages.js

```diff
@@ -0,0 +1,74 @@
+/* eslint-disable max-len */
+/**
+ * @flow
+ * @file i18n messages from Box UI Elements
+ * @author Box
+ */
+
+import { defineMessages } from 'react-intl';
+import type { MessageDescriptor } from 'react-intl';
+
+const messages: { [string]: MessageDescriptor } = defineMessages({
```

Comment:
Do these get translated properly by the build?

Reply:
I think so? I can double check in the commit where I use these components. The translation files are created on build